string node_name
string program_name
uint32 flags
uint8 id
uint8[6] joints_to_move
float64[6] joints_position
float64[6] joints_velocity